onnx/defs/nn/defs.cc

```diff
@@ -98,7 +112,7 @@ The convolution operator consumes an input tensor and {filter_desc}, and
 computes the output.)DOC";
             ReplaceAll(doc, "{filter_desc}", filter_desc);
             schema.SetDoc(doc);
-            schema.NumInputs(2, 3);
```

Comment:
I agree with this change, but please point it out in the PR, it's so easy to overlook.